src/state.js

```diff
@@ -0,0 +1,73 @@
+import Project from './project'
+import Task from './task'
+
+let db = firebase.firestore()
+const docRef = db.collection('projects').doc('Naqhid')
+
+export default function saveState() {
+  // localStorage.setItem('projects', JSON.stringify(window.projects));
+  docRef
+    .withConverter(projectConverter)
+    .set(window.projects)
```

Comment:
Do not use Window.alert() or Window.confirm(). It’s ugly, impossible to style, stop code execution and display differently on different browsers. Use custom modal instead


The previous reviewer @rindrajosia meant you should remove `window.`